hack/git/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
We generally use the `hack` directory for scripts that are actually used in CI and/or called from the `Makefile`.

Thinking that the `contrib/` directory would be a better place to put this 🤔